src/components/board/SharingTabSidebard.vue

```diff
@@ -68,7 +69,24 @@ export default {
 	computed: {
 		...mapGetters({
 			sharees: 'sharees'
-		})
+		}),
+		unallocatedSharees() {
+			let ret = []
+
+			let allocatedSharees = []
+			for (var user in this.board.acl) {
+				allocatedSharees.push(this.board.acl[user].participant.uid)
+			}
+
+			this.sharees.forEach(function(sharee) {
```

Comment:
This could be a bit simplified by using the filter method https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Maybe also get rid of the additional allocatedSharees array by using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex to check if the sharee is part of the board acl list.